src/models/Price.php

```diff
@@ -198,6 +198,11 @@ public function isOveruse()
         return strpos($this->type, 'overuse,') === 0;
     }
 
+    public function isOveruse95Traf()
```

Comment:
```suggestion
    public function isServer95Traf()
```
переименуй в других местах тоже